buildpipeline/dotnet-standard-win.json

```diff
@@ -55,15 +86,58 @@
         "nuGetAdditionalArgs": "-Timeout 3600",
         "verbosity": "Detailed",
         "nuGetVersion": "3.5.0.1829",
-        "nuGetPath": ""
+        "nuGetPath": "",
+        "continueOnEmptyNupkgMatch": "false"
+      }
+    },
+    {
+      "environment": {},
```

Comment:
Do you want to condition the publishing at all?
For example (from corefx repo)

```
"condition": "and(succeeded(), contains(variables.PB_PublishType, 'myget'), eq(variables.PB_ConfigurationGroup, 'Release'))",
```

Reply:
I don't think so, Standard doesn't pass a publishtype, and it appears we only build Release